clang/docs/ClangFormatStyleOptions.rst

```diff
@@ -5204,22 +5204,46 @@ the configuration (without a prefix: ``Auto``).
 
 .. _ReflowComments:
 
-**ReflowComments** (``Boolean``) :versionbadge:`clang-format 3.8` :ref:`¶ <ReflowComments>`
-  If ``true``, clang-format will attempt to re-flow comments. That is it
-  will touch a comment and *reflow* long comments into new lines, trying to
-  obey the ``ColumnLimit``.
+**ReflowComments** (``ReflowCommentsStyle``) :versionbadge:`clang-format 3.8` :ref:`¶ <ReflowComments>`
+  Comment reformatting style.
 
-  .. code-block:: c++
+  Possible values:
+
+  * ``// clang-format on`` (in configuration: ``// clang-format on``)
```

Comment:
is this correct?

Reply:
lol, no, I hadn't noticed it. All I did was `cd clang/docs/tools; ./dump_format_style.py`, following the recommendation in the script's header comment... but I think I know what's happening: I've moved the `clang-format` enabling and disabling comments inside `clang/include/clang/Format/Format.h`, and `dump_format_style.py` can't seem to cope with that...

Hah, yeah, it's a special case: [double-slash comments are interpreted as `config`](https://github.com/llvm/llvm-project/blob/159281a325841575b519dbf93bf0f981c319157e/clang/docs/tools/dump_format_style.py#L433), and printed like that. I don't know what that's used for, I think I'll simply move my `clang-format off` and `clang-format on` comments outside of the `enum` rather than disturb that functionality.

Reply:
Done, thank you for catching this, @mydeveloperday, and sorry I didn't notice it myself :sweat_smile:.